src/schedext-window.cpp

```diff
@@ -167,35 +198,47 @@ void SchedExtWindow::on_apply() noexcept {
         return "restart"sv;
     }();
 
-    static constexpr auto is_flags_commented = []() -> bool {
-        using namespace std::string_view_literals;
-        static constexpr auto scx_conf_path = "/etc/default/scx"sv;
-        const auto& scx_conf_content        = utils::read_whole_file(scx_conf_path);
-        return scx_conf_content.find("#SCX_FLAGS="sv) != std::string::npos;
-    };
-    static constexpr auto get_scx_flags_sed = [](std::string_view sched_flags_text, bool flags_commented) -> std::string {
+    static constexpr auto get_scx_flags_sed = [](std::string_view scx_sched,
+                                                 std::string_view scx_profile,
+                                                 std::string_view scx_extra_flags) -> std::string {
         using namespace std::string_literals;
-        if (sched_flags_text.empty() && !flags_commented) {
-            // comment out flags in scx
-            return "-e 's/SCX_FLAGS=/#SCX_FLAGS=/'"s;
-        } else if (!sched_flags_text.empty() && flags_commented) {
-            // set flags in scx
-            return fmt::format(R"(-e "s/.*SCX_FLAGS=.*/SCX_FLAGS='{}'/")", sched_flags_text);
-        } else if (!sched_flags_text.empty() && !flags_commented) {
-            // set flags in scx
-            return fmt::format(R"(-e "s/SCX_FLAGS=.*/SCX_FLAGS='{}'/")", sched_flags_text);
+
+        std::string scx_base_flags;
+
+	// Map the selected performance profile to the different scheduler
+	// options.
+	//
+        // NOTE: only scx_bpfland and scx_lavd are supported for now.
+        if (scx_profile == "default") {
+	} else if (scx_profile == "gaming") {
+            if (scx_sched == "scx_bpfland")
+                scx_base_flags = "-c 0 -k -m performance";
```

Comment:
```suggestion
                scx_base_flags = "-c 0 -k -m performance"sv;
```

in other places in that lambda also